### ISSUE TITLE
Enhancement Request: Include part-number in Multipart List Response

### ISSUE THREAD

**Issue by ashishpatel1992** [oracle/oci-cli#682]
Currently, when using the oci os multipart list command to list multipart uploads, there is no indication of the last successful part number uploaded. The response only includes the bucket, namespace, object, storage-tier, time-created, and upload-id as shown below:

```bash
oci os multipart list --bucket-name os_test -ns idgeqXXXXXX --profile OCI
{
  "data": [
    {
      "bucket": "os_test",
      "namespace": "idgeqXXXXXX",
      "object": "50MB.bin",
      "storage-tier": "Standard",
      "time-created": "2023-06-02T13:23:41.049000+00:00",
      "upload-id": "304f37e0-XXXX-XXXX-6603-31125bbXXXXXX"
    }
  ]
}
````
However, it would be highly beneficial if we could include the "part-number": 3, in the response to the request. This would provide valuable information about the upload progress, and would help users identify from which point they can resume their upload process if it was interrupted.

**Comment by kevco-us** [comment]
I agree that there's a gap in the cli's ability to show upload progress of an in-progress multipart upload.  

With the [ListMultipartUploadParts](https://docs.oracle.com/en-us/iaas/api/#/en/objectstorage/20160918/MultipartUpload/ListMultipartUploadParts) API docs I put together an `oci raw-request` to get details of an in-progress multipart upload.  Using details from your request above (and guessing the region for the uri):
```
oci raw-request \
   --http-method GET \
   --target-uri 'https://objectstorage.us-ashburn-1.oraclecloud.com/n/idgeqXXXXXX/b/os_test/u/50MB.bin?uploadId= 304f37e0-XXXX-XXXX-6603-31125bbXXXXXX'
```
In the URI I've substituted all of the bold components:  /n/**idgeqXXXXXX**/b/**os_test**/u/**50MB.bin**?uploadId=**304f37e0-XXXX-XXXX-6603-31125bbXXXXXX**
These plus the objectstorage.us-ashburn-1 would need to be updated to build a good request.
